README.md

```diff
@@ -63,7 +63,7 @@ docker-compose up -d
 
 Please note that it takes a while to start applications depending on how many ontologies you provided.
 
-After all ontologies are loaded, the viewer will be accessible from http://localhost:6201/.
+After all ontologies are loaded, the FIBO viewer will be accessible from http://localhost:6201/fibo/ontology.
```

Comment:
Maybe we can also add information about the dev view (http://localhost:6101/fibo/ontology`). It may be helpful for people. What do you think?

Reply:
I am not sure what you mean by:
> information about the dev view

Reply:
By the "dev view" I mean the view that is served by the module `https://github.com/edmcouncil/onto-viewer/tree/develop/onto-viewer-web-app` and not by the `html-pages`. And by the "information about the dev view" I meant simply that we can mention it here in the doc.

Reply:
File updated - could you review?